README.md

```diff
@@ -122,7 +122,7 @@ Do note that if you are planning to work with multiple Erlang versions on the sa
 
 ## Documentation
 
-Rebar3 documentation is maintained on [https://rebar3.org/docs](https://rebar3.org/docs)
+Rebar3 documentation is maintained on [https://hexdocs.pm/rebar3](https://hexdocs.pm/rebar3)
```

Comment:
I think long term we'll want to keep things on rebar3.org, but if necessary just proxy/redirect to hex.pm/rehost there. Keeping the domain and links that have been indexed and used for years working would be good.